crates/host/src/wasmbus/mod.rs

```diff
@@ -2374,12 +2481,13 @@ impl Host {
                     links
                 },
             );
-        let origin = WasmCloudEntity {
+        let origin: WasmCloudEntity = WasmCloudEntity {
```

Comment:
Why do we need to specify a type here now?

Reply:
Honestly I might have tab completed something on accident. I'll just remove